src/components/application_manager/rpc_plugins/app_service_rpc_plugin/test/app_service_plugin/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+include(${CMAKE_SOURCE_DIR}/tools/cmake/helpers/sources.cmake)
```

Comment:
@YarikMamykin I believe we should insert copyright comment into cmake files as well...

Reply:
@AKalinich-Luxoft fixed in 12854ed786c7b982ed2a5a553f560a4cad66dd8d